src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -139,6 +139,18 @@ public void addAuthor(Author author, List<String> ignoreGlobList) {
         propagateIgnoreGlobList(author, ignoreGlobList);
     }
 
+    /**
+     * Removes the authors in {@code ignoredAuthorsList}.
```

Comment:
This line is confusing to me.

Maybe something like the following or you can come out with a better one:
`Removes the authors provided in {@code ignoredAuthorsList} from {@code AuthorList}.`